### DISCUSSION TITLE
How to Summon Flying Mounts in a Custom Map?

### DISCUSSION THREAD

**Discussion by wbpzj1314** [azerothcore/azerothcore-wotlk#15763] [Guides & Tips]
I have joined a custom map, but I am currently unable to summon flying mounts directly in the new map. Although I have received a message saying that I cannot use flying mounts here, I can still use the GM command ".cast XXX" to summon one. However, I do not want to modify the mount skill flags in the spell.dbc file to enable flying, as this may have an impact on the official content.

I have also set the flag values for the map-related dbc files to be the same as those in Shattrath, but I still cannot summon flying mounts. Can you please advise me on how to modify the Azerothcore code to allow the map_id=1000 to summon flying mounts?

**Comment by Cassteriidoadw** [comment]
I also want to know
A good person lives a peaceful life

**Comment by pangolp** [comment]
If possible. Eh, even prohibit using them in areas where it should be possible. But they have to add a module or develop their own module using a hook that was added a while ago to the emulator. https://github.com/azerothcore/azerothcore-wotlk/pull/16590. The module that uses that functionality is this: but you can also create your own module if not. https://github.com/Tralenor/poc-can-fly-hook-override. Check that it is compatible with the latest version of the emulator. I don't know the status of the module, because I only tested it when the creator added the functionality and I didn't use it again.
